src/nautilus_librarian/_helper.py

```diff
@@ -0,0 +1,46 @@
+"""Get Version from pyproject.toml"""
+
+from pathlib import Path
+from typing import Any
+
+import atoml
+from atoml import TOMLDocument
+
+from _helpers import get_unicode_document, path_to_pyproject_toml
+
+
+def get_toml_document(string: str) -> TOMLDocument:
```

Comment:
maybe rename this function to: `parse_toml_doc_from_string` for additional clarity.